src/components/GraphFilter/GraphFind.tsx

```diff
@@ -100,10 +164,29 @@ export class GraphFind extends React.PureComponent<GraphFindProps> {
     this.handleFind();
   };
 
-  private handleFindSubmit = event => {
-    event.preventDefault();
-    this.findValue = this.findInputValue;
-    this.handleFind();
+  private handleFilter = () => {
+    if (!this.props.cyData) {
+      console.debug('Skip Filter: cy not set.');
+      return;
+    }
+    const cy = this.props.cyData.cyRef;
+    const selector = this.parseFindValue(this.filterValue);
+    cy.startBatch();
+    // this could also be done useing cy remove/restore but we had better results
+    // using visible/hidden (i.e. display: 'element'/'none' ).  The latter worked
+    // better when hiding animation, and also prevents the need for running
+    // layout after each update (at least in my tests so far..)
+    if (this.filteredElements) {
+      // make visible old filter-hits
+      this.filteredElements.style({ display: 'element' });
```

Comment:
One minor change, before recommending to use display: element / none i though that we did the layout to have a layout without the filtered stuff, this is not the case.
Lets change it to:
visibility: visible or visibility: hidden


![screenshot_2019-01-25_14-37-19](https://user-images.githubusercontent.com/3845764/51771596-22663500-20af-11e9-80ea-961f3f35ec27.png)


according to the docs[1]: 
> A display: none element does not take up space.
> A visibility: hidden element does take up space.

We definitely want it to take space, but be invisible

[1] http://js.cytoscape.org/#style/visibility

Reply:
@josejulio Yes, visibility is better for the styling.  We had to do some more work to get the surrounding edges set as well but it is fine. Thanks for your help!